src/main/java/com/epam/tc/hw3/pages/DifferentPage.java

```diff
@@ -0,0 +1,77 @@
+package com.epam.tc.hw3.pages;
+
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class DifferentPage {
+
+    public SoftAssertions softAssertions;
+
+    public DifferentPage(WebDriver driver) {
+        PageFactory.initElements(driver, this);
+        softAssertions = new SoftAssertions();
+    }
+
+    @FindBy(xpath = "//label[text()[contains(.,'Water')]]/input")
+    private WebElement checkBoxWater;
+
+    @FindBy(xpath = "//label[text()[contains(.,'Wind')]]/input")
+    private WebElement checkBoxWind;
+
+    @FindBy(xpath = "//label[text()[contains(.,'Selen')]]/input")
+    private WebElement radioSelen;
+
+    @FindBy(xpath = "//option[text() = 'Yellow']")
+    private WebElement dropDownYellow;
+
+    @FindBy(xpath = "//li[contains(text(), 'Water:')]")
+    private WebElement logForCheckBoxWater;
+
+    @FindBy(xpath = "//li[contains(text(), 'Wind:')]")
+    private  WebElement logForCheckBoxWind;
+
+    @FindBy(xpath = "//li[contains(text(), 'metal:')]")
+    private WebElement logForRadioSelen;
+
+    @FindBy(xpath = "//li[contains(text(), 'Colors:')]")
+    private WebElement logForDropDownYellow;
+
+    public void selectCheckBoxWater() {
+        checkBoxWater.click();
+    }
+
+    public void selectCheckBoxWind() {
+        checkBoxWind.click();
+    }
+
+    public void selectRadioSelen() {
+        radioSelen.click();
+    }
+
+    public void selectDropDownYellow() {
+        dropDownYellow.click();
+    }
+
+    public void verifySelectedCheckBoxWater() {
+        softAssertions.assertThat(logForCheckBoxWater.isDisplayed()).isTrue();
```

Comment:
isTrue is not needed here. True for all assertThat methods, which contains isTrue